src/renderer.ts

```diff
@@ -33,19 +33,29 @@ import LiveEventLink from 'components/liveEventLink';
 const getAttrs = (node: Node): Option<NamedNodeMap> =>
     isElement(node) ? some(node.attributes) : none;
 
-const getAttr = (attr: string) => (node: Node): Option<string> =>
+
+const transformHref = (href: string): string => {
+    if (href.startsWith('profile/')) {
+        return `https://www.theguardian.com/${href}`;
+    }
+
+    if (/https:\/\/www\.theguardian\.com\/[a-zA-Z0-9|/|-]+\/latest/.exec(href)?.length) {
```

Comment:
An alternative if you wanted to avoid the regex. Don't have a strong opinion either way, it's just regexes can be a bit brittle sometimes.

```ts
const url = new URL(href);
const path = url.pathname.split('/');
const isLatest = url.hostname === 'www.theguardian.com' && path[path.length - 1] === 'latest';
```

Reply:
This is about the third time you've suggested `URL` but I continue to think of regex... I think that's more readable

Reply:
Just a warning, the `URL` constructor can throw if it's not given a valid URL 🙄, so wrapping it in a `Result.fromUnsafe` might be helpful.

Reply:
yeah it probably ends up with more code, but it might be easier to handle future URLs that need transforming.